implementations/rust/ockam/ockam_identity/src/error.rs

```diff
@@ -1,17 +1,19 @@
+use ockam_core::compat::string::String;
 use ockam_core::{
     errcode::{Kind, Origin},
     Error,
 };
 
 /// Identity crate error
+#[repr(u8)]
```

Comment:
Why is that required?

Reply:
Because I have added a `String` to the `InvalidIdentifier` error.